src/main/java/com/unascribed/fabrication/mixin/f_balance/soul_speed_doesnt_damage_boots/MixinEnchantment.java

```diff
@@ -0,0 +1,38 @@
+package com.unascribed.fabrication.mixin.f_balance.soul_speed_doesnt_damage_boots;
+
+import com.unascribed.fabrication.FabConf;
+import com.unascribed.fabrication.support.EligibleIf;
+import com.unascribed.fabrication.support.injection.FabInject;
+import com.unascribed.fabrication.util.EnchantmentHelperHelper;
+import net.minecraft.component.ComponentMap;
+import net.minecraft.component.EnchantmentEffectComponentTypes;
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.registry.entry.RegistryEntryList;
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(Enchantment.class)
+@EligibleIf(configAvailable="*.soul_speed_doesnt_damage_boots")
+public class MixinEnchantment {
+
+	@Shadow
+	@Final
+	@Mutable
+	private ComponentMap effects;
+
+	@FabInject(at=@At("RETURN"), method="<init>")
+	private void modify(Text description, Enchantment.Definition definition, RegistryEntryList<Enchantment> exclusiveSet, ComponentMap effects, CallbackInfo ci) {
+		if (FabConf.isAnyEnabled("*.soul_speed_doesnt_damage_boots") && EnchantmentHelperHelper.matches(this, Enchantments.SOUL_SPEED)) {
```

Comment:
Should use `FabConf.isEnabled`
`sould speed doesn't damage boots` is not a extended feature, therefore this adds unececarry ovehead.
https://github.com/FalsehoodMC/Fabrication/blob/1d577f1cfcbff695c13d6f3365f97f8aca292807/features.d/h_balance.yml#L37

contrast this with any of the following:
https://github.com/search?q=repo%3AFalsehoodMC%2FFabrication%20%22extend%3A%22&type=code

`Creepers can't climb` extends `entities can't climb`.
so `FabConf.isAnyEnabled(entities can't climb)` translated to `FabConf.IsEnabled(entities can't climb) || FabConf.IsEnabled(Creepers can't climb)`